crates/cxx-qt-lib/build.rs

```diff
@@ -71,6 +71,7 @@ fn main() {
     };
 
     let mut rust_bridges = vec![
+        "core/qanystringview",
```

Comment:
guess there is also the question of whether it'll be used by other types in cxx-qt-lib or whether it should be in cxx-qt-lib-extras? It sounds like it is used by types already in cxx-qt-lib in Qt 6 though so would be useful ?

Reply:
I plan on implementing loadFromModule, which is part of QQmlApplication and thus should stay in cxx-qt-lib. That's under the assumption that  -lib can't depend on -extras, I haven't checked